opensearch/src/main/java/org/opensearch/sql/opensearch/storage/script/filter/lucene/RangeQuery.java

```diff
@@ -48,7 +56,8 @@ protected QueryBuilder doBuild(String fieldName, ExprType fieldType, ExprValue l
   }
 
   private Object value(ExprValue literal) {
-    if (literal.type().equals(ExprCoreType.TIMESTAMP)) {
+    if (literal.type().equals(ExprCoreType.TIMESTAMP)
+        && (fieldFormats == null || fieldFormats.isEmpty())) {
```

Comment:
why add this logic fieldFormats == null || fieldFormats.isEmpty()?

Reply:
Removed this.